public/app/features/alerting/unified/api/featureDiscoveryApi.ts

```diff
@@ -0,0 +1,19 @@
+import { AlertmanagerApiFeatures } from '../../../../types/unified-alerting-dto';
+
+import { alertingApi } from './alertingApi';
+import { discoverAlertmanagerFeatures } from './buildInfo';
+
+export const featureDiscoveryApi = alertingApi.injectEndpoints({
+  endpoints: (build) => ({
+    discoverAmFeatures: build.query<AlertmanagerApiFeatures, { amSourceName: string }>({
```

Comment:
One final nit: I think we should have fewer abbreviations since more folks will be helping us out on the front-end. How about `discoverAlertmanagerFeatures` in full?

Reply:
I'd like to but I have a bit of a clash with the `discoverAlertmanagerFeatures` function that comes from the `buildinfo.ts` file. I'm happy to improve the naming somehow